cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -103,7 +102,7 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
     }.value.unsafeRunSync.fold(error => throw new RuntimeException(s"broken parse! msg was $error"), identity)
 
     val wfd = wf.womExecutable match {
-      case Valid(Executable(wf: WorkflowDefinition)) => wf
+      case Right(Executable(wf: WorkflowDefinition)) => wf
       case o => fail(s"invalid executable $o")
```

Comment:
NYF: This case looks like it'll print out `invalid executable Left(blah)`, instead of just `invalid executable blah`

Reply:
Indeed!  I'm pretty sure this is endemic to my tests.